deployability/modules/testing/tests/helpers/agent.py

```diff
@@ -15,7 +15,7 @@ class WazuhAgent:
     @staticmethod
     def install_agent(inventory_path, agent_name, wazuh_version, wazuh_revision, live) -> None:
 
-        if live:
+        if live == "False":
             s3_url = 'packages'
             release = wazuh_version[:1] + ".x"
         else:
```

Comment:
If live, packages should be obtained from `packages.wazuh.com/4.x`, if not live, packages should be obtained from `packages-dev.wazuh.com/pre-release`, therefore, the if condition is wrong right?

Also, why are we using a string instead of a bool?